sdk/js/src/util/constants.js

```diff
@@ -0,0 +1,20 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const STACK_COMPONENTS = [ 'as', 'is', 'ns', 'js', 'gs' ]
+
+export {
+  STACK_COMPONENTS as default,
+  STACK_COMPONENTS,
+}
```

Comment:
i guess this is done on purspose to avoid `eslint` warnings?

Reply:
Correct. Do you prefer an ignore comment?

Reply:
no, its fine

Reply:
Fixed.